### DISCUSSION TITLE
General Questions

### DISCUSSION THREAD

**Discussion by jt4000** [elsa-workflows/elsa-core#1371] [Q&A]
Hello!

1.
What is the purpose to use ELSA

2.
What benefit is it to use ELSA?

3.
What business value can ELSAprovide?

Thank you!

**Comment by sfmskywalker** [comment]
Hi @jt4000 ,

1. The purpose of Elsa is to make it easy for .NET developers to add workflow capabilities to their applications with ease.
2. Elsa is easy to use, it's free & open source and it makes you happy (when it works ;)). Having workflows in your application makes it easy to quickly change business rules and even entire business processes.
3. Good question. I don't know.

Let me know if you have any additional questions!
